_utils/spellcheck.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+v() { [ -n "$VERBOSE" ] && echo "$@" ; }
+
+if command -v hunspell > /dev/null; then
+    check_words() { hunspell -l; }
+elif command -v aspell > /dev/null; then
+    check_words() { aspell | grep '^[#&]' | cut -d' ' -f2; }
+else
+    echo "${0##*/}: No spell checker installed. Need hunspell or aspell." >&2
+    exit 1
+fi
+
+extract_words() {
+    pandoc -tjson "$@" | jq -r '
```

Comment:
No, it intentionally infers the format based on the extension of the argument, which in this case is actually html, not markdown.